lib/helpers.coffee

```diff
@@ -15,3 +15,11 @@ module.exports =
       camelizedAttr = property.replace /\-([a-z])/g, (a, b) -> b.toUpperCase()
       styleObject[camelizedAttr] = value
     styleObject
+
+  ensureOpaqueBackground: (styleObject) ->
```

Comment:
It's from this PR: https://github.com/atom/tree-view/pull/565 ; I accidentally merged it in at some point. I can rebase it out if needed.